modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -378,22 +386,22 @@ Do not use these parameters with the `-I` or `S` parameters to configure REST AP
 `--amc-host`
 | `_amc-host_`
 
-|Specifies the service location URL of your local instance of Runtime Manager, such as `+https://10.0.0.1:8080/hybrid/v1+`.
+|Specifies the service location URL of your local instance of Runtime Manager, such as `+https://10.0.0.1:443/hybrid/v1+`.
 
 You can test whether the service is available at `_AMC_HOST_/hybrid/v1`.
 |`-W`
 
 `--mcm-host`
 | `_mcm-host_`
-|Specifies the service location URL of your local instance of MCM: for example, `wss://10.0.0.2:443/mule`.
+|Specifies the service location URL of your local instance of MCM: for example, `wss://10.0.0.2:8889/mule`.
 
 You can test whether the service is available at `_MCM_HOST_/mule`.
 
 |`-C`
 
 `--cs-host`
 |`_core-services-host_`
-|Specifies the service URL of your local instance of Access Management, for example, `+https://10.0.0.3:8080/accounts+`.
+|Specifies the service URL of your local instance of Access Management, for example, `+https://10.0.0.3:443/accounts+`.
```

Comment:
Should "the service URL" be "service location URL", as is the case at lines 389, 396, and 416?